megamek/src/megamek/client/ui/swing/boardview/EntitySprite.java

```diff
@@ -664,6 +687,21 @@ public void prepare() {
         graph.dispose();
     }
 
+    /** 
+     * Returns true when an indicator should be shown that some of the stacked
+     * units in the hex of this entity's may still move even while the topmost
+     * shown unit may already have moved. 
+     */
+    private boolean shouldIndicateNotDone() {
+        for (var other: bv.game.getEntitiesVector(entity.getPosition())) {
```

Comment:
```suggestion
        for (var other : bv.game.getEntitiesVector(entity.getPosition())) {
```
The spacing is more accessible text, as it makes it into a separate word.

Reply:
Might also be able to use a stream to check if any fit the check